### PR TITLE
add commas

### PR DESCRIPTION
**Pull request by Rnbsov**

### CODE REVIEW

**Comment by iliakan** [comment]
Are you 100% sure you're correct?

Comma after "in practice" can be omitted.

I also doubt about some other places. 

Please correct the syntax if you're 100% sure (preferably if you have native English or was educated well in syntax).

**Comment by Rnbsov** [comment]
Soo... I think I left only the necessary

**Comment by Rnbsov** [comment]
Before that, I consulted a little with native speakers

File: 1-js/05-data-types/04-array/article.md [hunk 1]
```diff
@@ -153,9 +153,9 @@ A stack is usually illustrated as a pack of cards: new cards are added to the to
 
 For stacks, the latest pushed item is received first, that's also called LIFO (Last-In-First-Out) principle. For queues, we have FIFO (First-In-First-Out).
 
-Arrays in JavaScript can work both as a queue and as a stack. They allow you to add/remove elements both to/from the beginning or the end.
+Arrays in JavaScript can work both as a queue and as a stack. They allow you to add/remove elements, both to/from the beginning or the end.
 
-In computer science the data structure that allows this, is called [deque](https://en.wikipedia.org/wiki/Double-ended_queue).
+In computer science, the data structure that allows this, is called [deque](https://en.wikipedia.org/wiki/Double-ended_queue).
```
Comment (smith558) [inline 1-js/05-data-types/04-array/article.md:158]: Remove the 2nd comma?

**Pull request merged**